docs/topics/null-safety.md

```diff
@@ -143,6 +143,17 @@ is `null`, the assignment is skipped and the expression on the right is not eval
 person?.department?.head = managersPool.getManager()
 ```
 
+## Nullable receiver
+
+Functions could be extended to use a [nullable receiver](extensions.md#nullable-receiver).
+This way you can check for the null condition and have a default defined in one place. 
+
+For example, the [`toString()` function](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/to-string.html) can be called with a nullable receiver and it returns the "null" string value (not `null`):
```

Comment:
“Can be called with a nullable receiver” sounds like it’s a choice the user makes. “The toString function is defined on a nullable receiver” is probably more appropriate.
Also maybe something like
“…for null values, it returns the String ‘null’” to make clear when that behaviour happens.

Reply:
[dee3b18](https://github.com/JetBrains/kotlin-web-site/pull/3133/commits/dee3b18d426b6f05ae9851974e57c3c4b955e3e6)